stats/gRPC.md

```diff
@@ -51,6 +51,8 @@ All client metrics should be tagged with the following.
 metrics. 
 `grpc_client_status` is set when an outgoing request finishes and is only available around metrics
 recorded at the end of the outgoing request.
+Status codes should be stringified according to:
+https://github.com/grpc/grpc/blob/ee6e4dc/doc/statuscodes.md
```

Comment:
Is there a way to link to the latest status code documentation?  That might be better than a specific commit, since the description could be updated.

Reply:
Pointed to master